servicemanager/servicemanager.go

```diff
@@ -181,6 +188,14 @@ func New(
 		return nil, aoserrors.Wrap(err)
 	}
 
+	sm.ttlStopChannel = make(chan bool, 1)
+
+	if err := sm.removeCachedServices(); err != nil {
+		return nil, err
```

Comment:
It's a good question, but I decide to return an error because all flow in the `New` function returns an error. But we could log Error and continue this function. What do you think?

Reply:
Done

Reply:
from my point of view we have to just log error. @al1img what do you think about it?